LibreNMS/Util/Mail.php

```diff
@@ -119,6 +119,12 @@ public static function send($emails, $subject, $message, bool $html = false, boo
                     $mail->Password = Config::get('email_smtp_password');
                     $mail->SMTPAutoTLS = Config::get('email_auto_tls');
                     $mail->SMTPDebug = 0;
+                    $mail->SMTPOptions = array(
+                        'ssl' => array(
+                            'verify_peer' => Config::get('email_smtp_verifypeer'),
```

Comment:
Please configure the detaults to be off :

`Config::get('email_smtp_verifypeer', false)`

Also, please use `[]` for arrays and not `array()`

Reply:
https://github.styleci.io/analyses/GPYL92?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link